### PR TITLE
PMOD-798 Create a json wikipedia Docker image

### PR DESCRIPTION
**Pull request by stathischaritos**

### CODE REVIEW

**Comment by dav009** [comment]
👍  it looks good

**Review by gphillips8frw — APPROVED** [review]

**Review by dav009 — APPROVED** [review]
👍

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+RUN apt-get update
+RUN yes | apt-get install -y build-essential maven
+
+COPY Makefile ./
+COPY VERSION ./
+COPY src ./src
+COPY pom.xml ./
+RUN make build
+ENTRYPOINT ["/var/lib/spark/bin/spark-submit"]
```
Comment (dav009) [inline Dockerfile:14]: I think the entrypoint should hide details to the caller.
Right now if I want to use this container I need to know where is the built jar located within the container.
a caller should not be aware of that, a caller concern should only be giving the input and output locations. 

We will need to fix this later anyway to provide a bootstrap script for AZ and to handle upload/downloads from cloud paths so I am happy to tick as it is now

Comment (tgalery) [reply Dockerfile:14]: what he said

**Review by tgalery — APPROVED** [review]

**Pull request merged**